package.json

```diff
@@ -17,10 +17,6 @@
     "coverage-publish": "aegir coverage publish",
     "docs": "aegir docs"
   },
-  "pre-commit": [
```

Comment:
This should not be removed, but changed to `pre-push`. I'd also like to see a better commit message :)

Reply:
@VictorBjelkholm can you handle the changes?